docker-compose.yml

```
@@ -6,5 +6,7 @@ services:
       context: .
     command: hypercorn -b 0.0.0.0:37888 quart_manual_run:application
     ports:
-      - "37888:37888"
-      - "5432:5432"
+      - 37888:37888
+      - 5432:5432
```

PyCharm raised a warning about it not being in double quotation marks.
```suggestion
      - "37888:37888"
      - "5432:5432"
```